src/dsp_tools/utils/id_to_iri.py

```diff
@@ -1,90 +1,156 @@
-"""
-This module handles the replacement of internal IDs with their corresponding IRIs from DSP.
-"""
```

Comment:
This module-level docstrings is superfluous. Important are the docstrings of the functions